app/lib/panoptes_api.rb

```diff
@@ -1,7 +1,6 @@
-require 'authenticator'
 require 'panoptes/client'
 
-class PanoptesClient
+class PanoptesApi
   attr_accessor :client
```

Comment:
You can add a delegator in here to forward methods to the `client`
```
delegate %i(authenticated_admin? authenticated_user_id), to: :client
```

Reply:
This is a great idea and makes the whole thing easier to test and use. 🥇